src/pkg/cli/compose/validation.go

```diff
@@ -28,14 +27,22 @@ func (e ErrMissingConfig) Error() string {
 	return fmt.Sprintf("missing configs %q (https://docs.defang.io/docs/concepts/configuration)", ([]string)(e))
 }
 
+type ErrManagedStoreParam string
```

Comment:
There's no longer any use of this, so I suggest to remove it and stick to regular `error`: `errors.New` or `fmt.Errorf`.